README.md

```diff
@@ -14,15 +14,16 @@ An all-in-one app for your Magic: the Gathering needs!
 - Delete unwanted cards from the trade.
 
 ## Counter
-- Keep track of the life totals of up to four players.
+- Keeps track of the life totals of up to four players.
```

Comment:
I feel like the rest of the descriptions use first person (e.g "<I> keep" vs "<the app> keeps"). I'm down to switch the whole thing to 3rd person but maybe we want to keep it first person until the whole read me uses 3rd? Or at least change it section by section.

Reply:
I mean, it's entirely up to you! Whatever you think sounds best I'll work off of that. I do tend to drift toward 3rd person because it helps focus the sentence action.

Reply:
Let's leave it for now, then when you revamp the whole section you can do it in 3rd.

Reply:
heard